src/components/useFetch.js

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from 'react';
+
+
+const useFetch = (callback, url) => {
+
+    const [loading,setLoading]=useState(false);
```

Comment:
```suggestion
    const [loading,setLoading] = useState(false);
```
지금 eslint와 prettier  설정이 안되어 있는 것 같아요!!